test/parse.jl

```diff
@@ -414,6 +414,26 @@ test_parseerror("0x1.0p", "invalid numeric constant \"0x1.0\"")
               try = "No"
            """)) == Expr(:error, "unexpected \"=\"")
 
+# issue #19861 make sure macro-expansion happens in the newest world for top-level expression
+@test eval(Base.parse_input_line("""
```

Comment:
nah, the `include_string` code-path was fine. this is an an entirely different bit of code.